package/support/package_darwin.sh

```diff
@@ -197,18 +173,30 @@ if [ "${SIGN_CODE}" = "1" ]; then
 </dict>
 </plist>
 EOF
+
     echo "Validating plist format..."
     plutil -lint entitlements.plist ||
         fail "Entitlements plist file format is invalid"
+
     echo "Signing all substrate executables..."
-    find "${SUBSTRATE_DIR}" -type f -perm +0111 -exec codesign --timestamp --options=runtime --entitlements entitlements.plist -s "${CODE_SIGN_IDENTITY}" {} \; ||
-        fail "Failure while signing executables"
+    for f in $( find "${SUBSTRATE_DIR}"  -type f -perm +0111 ); do
```

Comment:
Okay, but this will still fail if a path shows up with a space in it. I dunno why it split properly locally using the quoted command and `for` loop, but using `read` will be the safest way:

```sh
while IFS= read -rd '' f; do
...
done < <( find "${SUBSTRATE_DIR}"  -type f -perm +0111 -print0 )
```